stylesheets/components/avatar.css

```diff
@@ -0,0 +1,74 @@
+@import url("/stylesheets/main.css");
+
+.avatar {
+  margin: 1rem;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 100%;
+  position: relative;
+}
+
+.avatar-sm {
+  width: 2rem;
+  height: 2rem;
+}
+
+.avatar-lg {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+.avatar-xl {
+  width:4rem;
+  height:4rem;
+}
+
+.avatar-sq {
+  border-radius: 0.2rem;
+}
+
+.avatar-container {
+  position: relative;
+  width: fit-content;
+}
+
+.badge-online {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color: var(--badge-online);
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
+  border: 1.5px solid white;
+}
+
+.badge-idle {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color: var(--badge-idle);
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
+  border: 1.5px solid white;
+}
+
+.badge-dnd {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color:var(--badge-dnd) ;
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
```

Comment:
Except for the background-color, all the other styles are being repeated in all the badges, so its better to extract them in a separate class

Reply:
I have done as suggested.